content/en/developers/integrations/log_pipeline.md

```diff
@@ -16,111 +16,165 @@ further_reading:
   tag: Documentation
   text: Learn about log pipelines
 title: Create a Log Pipeline
-description: Learn how to create a Datadog Log integration.
+description: Learn how to create a Datadog Log Integration Pipeline.
 ---
 ## Overview
 
-This page walks Technology Partners through creating a log pipeline. A log pipeline is required if your integration is sending in logs. 
+This page walks Technology Partners through creating a log pipeline. A log pipeline is required if your integration sends logs to Datadog.
 
-## Log integrations
+## Integration prerequisites
 
-Use the [Logs Ingestion HTTP endpoint][1] to send logs to Datadog.
-
-## Development process
+In developing your integration to send logs to Datadog follow these guidlines to ensure the best experience for your users.
```

Comment:
guidelines*

Would suggest:

When developing your integration to send logs to Datadog, follow these guidelines to ensure the best experience for your users.